eng/pipelines/typespec-ci.yml

```diff
@@ -20,9 +20,19 @@ pr:
 
 jobs:
 - job: Validate_All_Specs
+
+  strategy:
+    matrix:
+      Linux:
+        Pool: azsdk-pool-mms-ubuntu-2204-general
+        OsVmImage: ubuntu-22.04
+      Windows:
```

Comment:
@mikeharder do we really think it is worth running this on both OS's all the time? I know there are some odd line-ending issues that we are working through, but I don't think that should have us run these against both every single PR.

Reply:
This is the CI pipeline which only runs on CI and scheduled builds, not PRs.  We need at least a nightly signal that development in the repo works on Windows.

Reply:
Cool yeah I'm good with the CI one just didn't think it was worth it for PR validation.

Reply:
Agree it's not worth it for PR validation.